app/containers/message/Urls.tsx

```diff
@@ -48,21 +48,51 @@ const styles = StyleSheet.create({
 		height: 150,
 		borderTopLeftRadius: 4,
 		borderTopRightRadius: 4
+	},
+	imageWithoutContent: {
+		borderRadius: 4
+	},
+	activityPosition: {
+		zIndex: 2,
+		position: 'absolute'
+	},
+	loadingState: {
+		height: 0,
+		borderWidth: 0
 	}
 });
 
 const UrlImage = React.memo(
-	({ image }: { image: string }) => {
+	({
+		image,
+		setImageLoadedState,
+		style
+	}: {
+		image: string;
+		setImageLoadedState(value: TImageLoadedState): void;
+		style: StyleProp<ImageStyle>;
+	}) => {
 		const { baseUrl, user } = useContext(MessageContext);
 
 		if (!image) {
 			return null;
 		}
 
 		image = image.includes('http') ? image : `${baseUrl}/${image}?rc_uid=${user.id}&rc_token=${user.token}`;
-		return <FastImage source={{ uri: image }} style={styles.image} resizeMode={FastImage.resizeMode.cover} />;
+
+		return (
+			<>
+				<FastImage
+					source={{ uri: image }}
+					style={[styles.image, style]}
```

Comment:
Both components are on the same file.
Why do you need to send style as a prop?
This is what's breaking memo.

Reply:
Instead of pass the style or pass hasContent and imageLoadedState to the UrlImage, I merged the UrlImage inside the URL, as you talked here: https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/4723#discussion_r1107538174